packages/tpetra/core/compat/Tpetra_KokkosCompat_ClassicNodeAPI_Wrapper.hpp

```diff
@@ -51,14 +51,8 @@ class KokkosDeviceWrapperNode {
   static constexpr bool is_serial = false;
 #endif
 
-  //! Whether the ExecutionSpace is CPU-like (its default memory space is HostSpace or HBWSpace)
```

Comment:
I was a bit hasty, you might want to keep the comment with some update.

Reply:
It's fine, the comment doesn't explain anything that the actual code doesn't :)